src/Hyperion/Main.hs

```diff
@@ -78,7 +81,8 @@ options = do
          (Options.strOption
             (Options.long "output" <>
              Options.short 'o' <>
-             Options.metavar "FILENAME"))
+             Options.help "Outputs the benchark to the directory" <>
```

Comment:
It is kinda our problem because if the user does use `stack` this will not work, and *we* use stack.

The behaviour is what i understood of what you said, just wanted to meke sure that we understood each other.